### PR TITLE
Initial release notes for NSM v1.1.1

### PR DESCRIPTION
**Pull request by edwarnicke**
Signed-off-by: Ed Warnicke <hagbard@gmail.com>

### CODE REVIEW

File: content/docs/releases/v1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+short = "v1.1.0"
+weight = 1
+date = "2021-11-28"
++++
+
+NSM v1.1.1 is released via a set of [example use cases](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.1.1).
+
+## Kubernetes Compatibility
+NSM v1.1.1 has been successfully [integration tested](https://github.com/networkservicemesh/integration-k8s-kind/actions/runs/1514862151) with Kubernetes versions:
+
```
Comment (denis-tingaikin) [inline content/docs/releases/v1.1.0.md:12]: I think we need to point that it is related to _server_ k8s versions.

File: content/docs/releases/v1.1.0.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+- [Network Service Mesh Interdomain](http://localhost:1313/docs/concepts/architecture/#inter-domain) now supports Spire Federation.   Spire Federation allows workloads from
+different Trust Domains to connect to a Network Service securely and subject to authorization.
+- [Network Service Mesh Interdomain CI](https://github.com/networkservicemesh/integration-interdomain-k8s) now tests interdomain simultaneously across:
+    - GKE
+    - EKS
+    - AKS
+    - Vanilla K8s on Bare Metal (using Equinix Metal)
+   validating that, for example, a Pod in GKE can connect to a Network Service being provided by a Pod in AKS.
+
```
Comment (denis-tingaikin) [inline content/docs/releases/v1.1.0.md:55]: Do we also need to add that we tested release on interdomain ci?
```suggestion
NSM v1.1.1 has been successfully [integration interdomain tested](https://github.com/networkservicemesh/integration-interdomain-k8s/actions/runs/1514861946)
```

File: content/docs/releases/v1.1.0.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+As the number and variety of NSM use cases has grown, some use cases have emerged in which the Forwarder needs to do more complex cross connects than just P2P.  Point to multipoint (P2MP), by contrast to P2P, have a large number of mutually incompatible implementations.  Often P2MP will be spoken of in shorthand as if its is a single unitary thing: it is not.
+
+In order to support these sorts of use cases, NSM made a slight architectual enhancement: have the forwarder select the NSE (or in the case of P2MP NSEs) to which it should be connecting.  This change allows a Forwarder to support zero or more possible P2MP senarios.
+
+Given a Network Service that requires some particular P2MP capabilty (or any other kind of capability) from its Forwarder for one of its logical links, it is necessary to have some capacity of 'hinting' that requirement to the NSMgr so that it selects the correct Forwarder in the event that more than on is present.  NetworkService matches can now be decorated with 'metadata' with 'labels' for use by the NSMgr in selecting the correct Forwarder for each logical hop the match represents.
+
+
+### Interdomain improvements
```
Comment (denis-tingaikin) [inline content/docs/releases/v1.1.0.md:45]: Hmm, I think we also can point at an example to get started to run interdomain.
Something like
```
(Try it)[https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/interdomain#interdomain]
```

File: content/docs/releases/v1.1.0.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+
+### OVS Forwarder
+A new forwarder based on the OpenvSwitch (OVS) data plane has been added to NSM as alternative to the VPP forwarder. The OVS forwarder can install its own OVS instance or re-use an existing OVS instance on the host. For standard use cases, the OVS forwarder supports the local kernel mechanism (using veth pairs) and remote VXLAN mechanism with typical kernel datapath performance.
+
+For high-performance use cases the OVS forwarder can off-load the NSM data plane to a Mellanox ConnectX SmartNIC, relying on standard OVS HW-offload capabilities. In this scenario, the OVS forwarder injects ConnectX (smart) Virtual Functions into NSC and NSE pods as local mechanism, where they can be consumed as DPDK or RDMA user-space interfaces to achieve full pass-through performance. The SmartNIC handles bonding and VXLAN encapsulation transparently for the applications.
+
+The OVS forwarder is also planned to support coming forwarder use cases, such as remote VLAN mechanism for break-out to external fabric, VLAN-trunking of multiple client connections on a single NSE interface and P2MP forwarding.
+
```
Comment (denis-tingaikin) [inline content/docs/releases/v1.1.0.md:70]: We also can point at our OVS examples:
```
[Try it](https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/ovs)
```

File: content/docs/releases/v1.1.0.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
++++
+title = "Release v1.1.0"
```
Comment (Sol-0) [inline content/docs/releases/v1.1.0.md:2]: Shouldn't the version be "v1.1.1" here?

File: content/docs/releases/v1.1.0.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+### VPP Forwarder Improvements
+
+- SRIOV support incorporated into the [VPP Forwarder](https://github.com/networkservicemesh/cmd-forwarder-vpp)
+  Pods can now request either a 'kernel' or 'vfio' interface backed by an SRIOV VF that connects it to its requested Network Service be injected into the Pod
+- Vlan remote mechanism: the [VPP Forwarder](https://github.com/networkservicemesh/cmd-forwarder-vpp) now supports use of vlan as a remote mechanism.
+  This allows the physical network to provide a Network Service, and distinguish between vWires by vlan tag.  Please note this feature is alpha as CI driven examples are not yet available.
+
+
+### OVS Forwarder
+A new forwarder based on the OpenvSwitch (OVS) data plane has been added to NSM as alternative to the VPP forwarder. The OVS forwarder can install its own OVS instance or re-use an existing OVS instance on the host. For standard use cases, the OVS forwarder supports the local kernel mechanism (using veth pairs) and remote VXLAN mechanism with typical kernel datapath performance.
```
Comment (pperiyasamy) [inline content/docs/releases/v1.1.0.md:65]: Can you also refer to ovs forwarder repo link https://github.com/networkservicemesh/cmd-forwarder-ovs ?

**Comment by JanScheurich** [comment]
Looks good to me!

File: content/docs/releases/v1.1.0.md (hunk 1 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md:12]: Could you say more, I don't quit follow.

File: content/docs/releases/v1.1.0.md (hunk 2 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md:56]: Does this work: https://github.com/networkservicemesh/site/pull/207/commits/99a0d4b0e59b6e8e723f2ee1a835d5f941610005 ?

File: content/docs/releases/v1.1.0.md (hunk 3 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md:45]: Does this work? https://github.com/networkservicemesh/site/pull/207/commits/e159baebc8a6873441e2c052214e9aaff92c188c

File: content/docs/releases/v1.1.0.md (hunk 4 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md:73]: Does this work?

File: content/docs/releases/v1.1.0.md (hunk 6 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md]: @pperiyasamy Does this work: https://github.com/networkservicemesh/site/pull/207/commits/fc8a5c610e0ab7481945bc73eabe2ae5f9269212

File: content/docs/releases/v1.1.0.md (hunk 5 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md:2]: This is mostly to keep with the previous pattern.

File: content/docs/releases/v1.1.0.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+- v1.21.1
+- v1.20.7
+- v1.19.11
+- v1.18.15
+- v1.17.11
+
+## Known Issues
+
+###   GKE Compatibility
+GKE has manifest an odd Netlink IP Neighbor bug with IPv6 in its latest node-version.  This is expected to worked around automatically in NSM in the future.  Meanwhile, there is a workaround:
```
Comment (fkautz) [inline content/docs/releases/v1.1.0.md:23]: Grammar here can be improved.

File: content/docs/releases/v1.1.0.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+For high-performance use cases the [OVS forwarder](https://github.com/networkservicemesh/cmd-forwarder-ovs) can off-load the NSM data plane to a Mellanox ConnectX SmartNIC, relying on standard OVS HW-offload capabilities. In this scenario, the OVS forwarder injects ConnectX (smart) Virtual Functions into NSC and NSE pods as local mechanism, where they can be consumed as DPDK or RDMA user-space interfaces to achieve full pass-through performance. The SmartNIC handles bonding and VXLAN encapsulation transparently for the applications.
+
+The [OVS forwarder](https://github.com/networkservicemesh/cmd-forwarder-ovs) is also planned to support coming forwarder use cases, such as remote VLAN mechanism for break-out to external fabric, VLAN-trunking of multiple client connections on a single NSE interface and P2MP forwarding.
```
Comment (fkautz) [inline content/docs/releases/v1.1.0.md:75]: I suspect "coming" is supposed to be "incoming"

File: content/docs/releases/v1.1.0.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+
+### Usability
+
+- Examples in [deployment-k8s](https://github.com/networkservicemesh/deployments-k8s/blob/v1.1.1/README.md) are greatly simplified by using [Spire K8s Registrar](https://github.com/spiffe/spire/blob/4e1a60a4d065f6a028f8c384c85603fbc151d5e8/support/k8s/k8s-workload-registrar/README.md) to create workload identities.
+- Kustomize examples in [deployment-k8s](https://github.com/networkservicemesh/deployments-k8s/blob/v1.1.1/README.md) can now be run with urls rather than inline in a checked out copy of the repo as was required in v1.0.0
+
+### Stability
+- Healing - simplified to improve stability and reduce reconvergence time
+- Refresh, and Timeout vastly simplified resulting in improved stability.
```
Comment (fkautz) [inline content/docs/releases/v1.1.0.md:87]: Remove the comma

**Review by fkautz — CHANGES_REQUESTED** [review]

File: content/docs/releases/v1.1.0.md (hunk 7 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md]: Fixed in https://github.com/networkservicemesh/site/pull/207/commits/a94b37d7475128c89c4e040f6aac0789ff094afa

File: content/docs/releases/v1.1.0.md (hunk 8 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md]: Fixed to "upcoming" in https://github.com/networkservicemesh/site/pull/207/commits/a94b37d7475128c89c4e040f6aac0789ff094afa

File: content/docs/releases/v1.1.0.md (hunk 9 above)
Comment (edwarnicke) [reply content/docs/releases/v1.1.0.md]: Fixed in https://github.com/networkservicemesh/site/pull/207/commits/a94b37d7475128c89c4e040f6aac0789ff094afa

**Review by denis-tingaikin — APPROVED** [review]
LGTM

**Comment by denis-tingaikin** [comment]
@fkautz , @pperiyasamy Let us know if you still have any open comments then we'll resolve them in separate PRs.

**Pull request merged**